broadcaster/message/message_serialization_test.go

```diff
@@ -13,7 +13,40 @@ import (
 	"github.com/offchainlabs/nitro/arbos/arbostypes"
 )
 
-func ExampleBroadcastMessage_broadcastfeedmessage() {
+func ExampleBroadcastMessage_broadcastfeedmessageWithBlockHash() {
+	var requestId common.Hash
+	msg := BroadcastMessage{
+		Version: 1,
+		Messages: []*BroadcastFeedMessage{
+			{
+				SequenceNumber: 12345,
+				Message: arbostypes.MessageWithMetadata{
+					Message: &arbostypes.L1IncomingMessage{
+						Header: &arbostypes.L1IncomingMessageHeader{
+							Kind:        0,
+							Poster:      [20]byte{},
+							BlockNumber: 0,
+							Timestamp:   0,
+							RequestId:   &requestId,
+							L1BaseFee:   big.NewInt(0),
+						},
+						L2msg: []byte{0xde, 0xad, 0xbe, 0xef},
+					},
+					DelayedMessagesRead: 3333,
+				},
+				BlockHash: &common.Hash{0: 0xff},
+				Signature: nil,
+			},
+		},
+	}
+	var buf bytes.Buffer
+	encoder := json.NewEncoder(&buf)
+	_ = encoder.Encode(msg)
+	fmt.Println(buf.String())
+	// Output: {"version":1,"messages":[{"sequenceNumber":12345,"message":{"message":{"header":{"kind":0,"sender":"0x0000000000000000000000000000000000000000","blockNumber":0,"timestamp":0,"requestId":"0x0000000000000000000000000000000000000000000000000000000000000000","baseFeeL1":0},"l2Msg":"3q2+7w=="},"delayedMessagesRead":3333},"blockHash":"0xff00000000000000000000000000000000000000000000000000000000000000","signature":null}]}
```

Comment:
Is it possible to make the test check the output?

Reply:
Yep, the Output is already checked: https://go.dev/blog/examples